repo/packages/Q/quobyte/0/resource.json

```diff
@@ -0,0 +1,19 @@
+{
+  "images": {
+    "icon-small": "http://www.quobyte.com/mesos/icon-quobyte-small.png",
+    "icon-medium": "http://www.quobyte.com/mesos/icon-quobyte-medium.png",
+    "icon-large": "http://www.quobyte.com/mesos/icon-quobyte-large.png",
+    "screenshots": [
+      "http://www.quobyte.com/mesos/screen-1.png",
+      "http://www.quobyte.com/mesos/screen-2.png",
+      "http://www.quobyte.com/mesos/screen-3.png"
+    ]
+  },
+  "assets": {
+    "container": {
+      "docker": {
+        "f8c203e427cc": "quobyte/quobyte-mesos"
```

Comment:
I based this on the documentation. Multiverse Readme points to the Universe Readme which states:

    For the Docker image, please use the image ID for the referenced image. You can find this by pulling the image locally and running docker images some-org/foo:1.0.0.

As a note: I'm not too sure this key is useful, too.